core/services/ocr2/delegate.go

```diff
@@ -895,9 +895,6 @@ func (d *Delegate) newServicesLLO(
 	if err != nil {
 		return nil, ErrJobSpecNoRelayer{Err: err, PluginName: "streams"}
 	}
-	if rid.Network != relay.NetworkEVM {
```

Comment:
Just to double-check, you don't need this log anymore?

Reply:
This was a check, not a log if I'm not mistaken, and there isn't really any reason for it to be here. LLO should support any relayer type.